lib/src/view/design_side_helix.dart

```diff
@@ -92,6 +94,14 @@ class DesignSideHelixComponent extends UiComponent2<DesignSideHelixProps> with P
               ? temp_text_for_help_doc_figure_making
               : props.helix.idx.toString(),
           Dom.svgTitle()(tooltip)),
+      (Dom.text()
+            ..style = {'font-size': 10}
+            ..dominantBaseline = 'text-before-edge'
+            ..textAnchor = 'middle'
+            ..y = '10'
```

Comment:
Instead of hard-coding the value 10 here, I'd specify it as a fraction of `constants.HELIX_RADIUS_SIDE_PIXELS`. That way if we change that constant later, the text will still go to the same relative position of the helix. (Of course, the font size is hard-coded and will look bigger or smaller relative to the helix, but we can deal with that in a separate issue where we let the user control the font size themselves.)